skstore/ts/src/skstore_api.ts

```diff
@@ -235,14 +235,28 @@ export interface NonEmptyIterator<T> {
 
 /**
  * A _Lazy_ Handle on a reactive collection, whose values are computed only when queried
- * using `get`
  */
 export interface LHandle<K extends TJSON, V extends TJSON> {
   /**
-   * Get (and potentially compute) a value of a lazy reactive collection
+   * Get (and potentially compute) all values mapped to by some key of a lazy reactive
+   * collection.
    * @throws {Error} when the key does not exist
    */
-  get(key: K): V;
+  getArray(key: K): V[];
+
+  /**
+   * Get (and potentially compute) a value of a lazy reactive collection.
+   * If multiple values are mapped to by the key, any of them can be returned
+   * @throws {Error} when the key does not exist
+   */
+  getFirst(key: K): V;
+
+  /**
+   * Get (and potentially compute) a value of a lazy reactive collection, if one exists.
+   * If multiple values are mapped to by the key, any of them can be returned.
+   * @throws {Error} when the key does not exist
+   */
+  maybeGetFirst(key: K): Opt<V>;
```

Comment:
Here it is also unclear what the difference between returning None vs throwing is.